### PR TITLE
RN: CNV-5273 secrets/ConfigMaps/service accounts

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-5273
Preview build: http://file.bos.redhat.com/pousley/071720/cnv-5273-rn/virt/virt-2-4-release-notes.html#virt-2-4-web-new

### CODE REVIEW

**Comment by gouyang** [comment]
LGTM

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -87,6 +87,7 @@ from being inadvertently started before an upload has completed.
 //CNV-5272 - Configure scheduling directly from CNV UI.
 
 //CNV-5273 - Attach a config map / secret / service account to a VM from the UI.
+* You can now add xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[secrets, ConfigMaps, and service accounts] to a virtual machine on the *Virtual Machine Overview* -> *Environment* page of the {product-title} web console. You can also remove these resources by using the same form.
```
Comment (aburdenthehand) [inline virt/virt-2-4-release-notes.adoc:90]: epic nit: 'the same form'?

**Comment by aburdenthehand** [comment]
One nit
otherwise LGTM

File: virt/virt-2-4-release-notes.adoc (hunk 1 above)
Comment (ousleyp) [reply virt/virt-2-4-release-notes.adoc:90]: I think I'll change it to "on the same page" to avoid ambiguity of meaning :)

**Pull request merged**